### ISSUE TITLE
Hack my old Instagram

### ISSUE THREAD

**Issue by julieduarte788** [SwetabhOfficial/instagram-hacking-tool#9]
I need help getting into my Instagram account? can you help me?

**Comment by dhanushkkarh** [comment]
Hello. Thanks for your interest in Instagram hacking tool. 
I understand that you are in need to recover your old account

However, i would like to let you know that Phishing Tools are nothing but Instagram Page Traps to lure users into entering their username and passwords which is then captured into our servers.
I recommend you searching for Brute force tools 

WhatsApp me on +916380552921

**Comment by heatherbuechele** [comment]
What was my old password again😢

**Comment by its-pablo** [comment]
> WhatsApp me on +916380552921

giving your phone number is generally not advised on public forums such as these

**Comment by Rohit7791** [comment]
https://www.instagram.com/_ishitasharma0

Please can someone tell the email id, number and location of this account?

**Comment by its-pablo** [comment]
@Rohit7791 
No. No one can tell you the Email Address, phone number or location of the account because nobody knows what it is.
Ask elsewhere, **please**
